.github/workflows/msal-node-e2e.yml

```diff
@@ -105,3 +105,95 @@ jobs:
       with:
         name: e2e-test-screenshots
         path: samples/**/screenshots
+
+  run-electron-e2e:
+    if: (github.repository == 'AzureAD/microsoft-authentication-library-for-js') && (github.actor != 'dependabot[bot]') && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest]
+        sample:
+          - 'ElectronTestApp'
+
+    name: ${{ matrix.sample }} - ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14
```

Comment:
Reason why we only need to test against one version of node? If we need to choose one I'd suggest 16 or 18 as we're going to start enforcing 16+ for local development of our repo soon

Reply:
Electron itself is packaged with a particular version of Node, so I didn't see much value in testing over a range of node versions. Let me switch to v16 in that case.